src/anomalib/pipelines/tiled_ensemble/components/__init__.py

```diff
@@ -0,0 +1,30 @@
+"""Tiled ensemble pipeline components."""
```

Comment:
And same for the other tiled ensemble implementations. It would be good to credit your work instead of adding an Intel license only